bigbluebutton-html5/imports/api/users/server/handlers/presenterAssigned.js

```diff
@@ -16,5 +16,12 @@ export default function handlePresenterAssigned({ body }, meetingId) {
   };
 
   const prevPresenter = Users.findOne(selector);
+
+  // no previous presenters
+  if (!prevPresenter) {
+    // return value is sponsored by eslint
```

Comment:
:joy:

Reply:
cant you just do a `return;` without value?

Reply:
Eslint complains.. shows it as an error :-|